test/burn/org/apache/cassandra/utils/LongBTreeTest.java

```diff
@@ -803,7 +803,7 @@ public void testLargeBatchesLargeRange() throws ExecutionException, InterruptedE
     public void testRandomRangeAndBatches() throws ExecutionException, InterruptedException
     {
         Random seedGenerator = new Random(randomSeed());
-        for (int i = 0 ; i < perThreadTrees / 10 ; i++)
+        for (int i = 0 ; i < perThreadTrees / 100 ; i++)
```

Comment:
I'm happy with this change but have an observation about it. The comments at the top of the class do say that all the variables should be randomised. It seems to me that an alternative approach here be to just use a random value for the number of iterations, say between 10 & 100.

Reply:
This loop controls the number of repetitions of the same test, which is to some degree randomized. Each iteration gets a different random seed. I think the top level documentation had different variables in mind, e.g. things like treeSize or number of threads etc.

Reply:
Fair enough, I'm happy with the change anyway. The number of times it runs is pretty arbitrary anyway.